test-app/.template-lintrc.js

```diff
@@ -2,4 +2,7 @@
 
 module.exports = {
   extends: 'recommended',
+  rules: {
+    'no-inline-styles': false,
```

Comment:
I disabled this rule because I think it doesn't make sense in tests.

Reply:
Agreed